### DISCUSSION TITLE
Consistent dropdown behavior with starring and watching

### DISCUSSION THREAD

**Discussion by MattIPv4** [community/community#8239] [Other Feature Feedback, Questions, & Ideas]
Really minor thing, but irked part of my brain -- the dropdown behaviour for watching is that there is no default and pressing any part of the button will trigger the dropdown. However, with starring, there is a default behaviour of just starring the repo, and the dropdown is a separate click trigger to the side of the button.

I like that starring is still the default behaviour and don't want this to change, but now that this dropdown exists, I almost think it'd be nice for the watching dropdown to have matching behaviour, where clicking on the majority of the button watches, and the dropdown is a secondary trigger to the side of the button.

**Comment by brianlovin** [comment]
Thanks for the suggestion @MattIPv4 — we've had a few other people mention that the split button can feel a bit weird. I think having it all combined into a single button shape but still having two clickable areas would be slightly more confusing, as people would likely mis-click. But, it feels like there's an underlying issue here that the different button behaviors in the repo header are confusing — I'll share with the team and think about better options here 😊
